lib/components/viewers/amenities-panel.tsx

```diff
@@ -114,13 +138,13 @@ class AmenitiesPanel extends Component {
               <>
                 <div>
```

Comment:
Can use `<br />` tags instead of `<divs>`, with the advantage that you can improve the text indentation using:
```css
.related-item {
    text-indent: -27px; /* 22px image width + 5px margin */
    padding-left: 27px;
}
```